draft-ietf-quic-transport.md

```diff
@@ -1749,12 +1749,13 @@ values for 0-RTT.  This includes initial_max_data and either
 initial_max_streams_bidi and initial_max_stream_data_bidi_remote, or
 initial_max_streams_uni and initial_max_stream_data_uni.
 
-It is OPTIONAL for a server to be able to recover the previously sent values of
-the max_idle_timout, max_udp_payload_size, and disable_active_migration
-parameters. Lowering the values of these parameters while also accepting 0-RTT
-data could degrade the performance of the connection. Specifically, lowering the
-max_udp_payload_size could result in dropped packets leading to worse
-performance compared to rejecting 0-RTT data outright.
+A server MAY choose to store and recover the previously sent values of the
+max_idle_timout, max_udp_payload_size, and disable_active_migration parameters
+and reject 0-RTT if it selects smaller values. Lowering the values of these
```

Comment:
This new phrase "reject 0-RTT if it selects smaller values" is confusing. What are you trying to convey?

Reply:
That was from MT's suggestion. It's saying how the server might use the remembered values. It might be redundant with the following sentence - I don't care either way whether it's here.

Reply:
"if the server chooses values for the resumed connection that are smaller than the original connection." ?

Reply:
I think the sentence is fine. I don't know why I was unable to parse it last time.